komga-webui/public/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Komga",
+  "short_name": "Komga",
+  "start_url": ".",
+  "display": "fullscreen",
+  "background_color": "#08397f",
+  "description": "Free and open source comics/mangas media server",
```

Comment:
I just ripped the description from the Komga website, could probably use a better one in here instead if you'd like